internal/venice-common/src/main/java/com/linkedin/venice/writer/CompositeVeniceWriter.java

```diff
@@ -23,13 +20,13 @@
 @NotThreadsafe
 public class CompositeVeniceWriter<K, V, U> extends AbstractVeniceWriter<K, V, U> {
   private final VeniceWriter<K, V, U> mainWriter;
-  private final VeniceWriter<K, V, U>[] childWriters;
+  private final AbstractVeniceWriter<K, V, U>[] childWriters;
```

Comment:
This class currently is only being used in VPJ, so shall we move it to `venice-push-job` module instead to avoid confusion?